Docker/directory-permission-setter/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:24.04
```

Comment:
There is not readme for this Docker file.

PLease add readme to explain the reason and how the image should be used. :).